packages/gatsby-transformer-pdf/src/gatsby-node.js

```diff
@@ -0,0 +1,49 @@
+const Promise = require(`bluebird`)
+const PDFParser = require(`pdf2json`)
+
+const convertToJson = path =>
+  new Promise((res, rej) => {
+    const pdfParser = new PDFParser(this, 1)
+    pdfParser.loadPDF(path)
+    pdfParser
+      .on(`pdfParser_dataReady`, pdfData => {
+        res(pdfParser.getRawTextContent())
```

Comment:
Only if you know off-hand, is it possible to get more structured data from this, or is a string representation of the content as good as we can get?

Reply:
well, i'm trying to get the content of each page line by line.